app/src/main/java/ngo/teog/swift/gui/main/BarcodeFragment.java

```diff
@@ -49,10 +51,12 @@
     private String lastText;
 
     private Button searchButton;
+    private ImageView torchButton;
     private EditText searchField;
     private ProgressBar progressBar;
+    private boolean torchState = false;
```

Comment:
variable name "torchState" should be changed -> make clear, what true/false mean